libs/components/packages/src/schematics/migrations/update-7/fix-imports.ts

```diff
@@ -6,49 +6,23 @@ import * as path from 'path';
 
 import { getWorkspace } from '../../utility/workspace';
 
-function updateTypescriptImports(
+const CORE_JS_UNNECESSARY =
+  /import \{ (Object|Date|setTimeout) } from 'core-js';\r?\n/;
```

Comment:
We shouldn't see any imports from `core-js`. Is there a reason you chose these three types? Can we remove all imports from `core-js`?